integration/php_httpd_test.go

```diff
@@ -78,6 +83,46 @@ func testPhpHttpd(t *testing.T, context spec.G, it spec.S) {
 			Expect(logs).To(ContainLines(ContainSubstring("Apache HTTP Server Buildpack")))
 			Expect(logs).To(ContainLines(ContainSubstring("PHP Web Buildpack")))
 			Expect(logs).To(ContainLines(ContainSubstring("PHP Composer Buildpack")))
+			Expect(logs).NotTo(ContainLines(ContainSubstring("Procfile Buildpack")))
+		})
+
+		context("when there is a Procfile", func() {
+			it.Before(func() {
+				Expect(ioutil.WriteFile(filepath.Join(source, "Procfile"), []byte("web: procmgr /layers/paketo-buildpacks_php-web/php-web/procs.yml"), 0644)).To(Succeed())
+			})
+
+			it("creates a working OCI image and uses the Procfile buildpack", func() {
+				var err error
+				var logs fmt.Stringer
+				image, logs, err = pack.WithNoColor().Build.
+					WithBuildpacks(phpBuildpack).
+					WithPullPolicy("never").
+					Execute(name, source)
+				Expect(err).NotTo(HaveOccurred(), logs.String())
+
+				container, err = docker.Container.Run.
+					WithEnv(map[string]string{"PORT": "8080"}).
+					WithPublish("8080").
+					WithPublishAll().
+					Execute(image.ID)
+				Expect(err).NotTo(HaveOccurred())
+
+				Eventually(container).Should(BeAvailableAndReady(), ContainerLogs(container.ID))
+
+				response, err := http.Get(fmt.Sprintf("http://localhost:%s", container.HostPort("8080")))
+				Expect(err).NotTo(HaveOccurred())
+				defer response.Body.Close()
+
+				content, err := ioutil.ReadAll(response.Body)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(string(content)).To(MatchRegexp("This is an HTTPD app."))
+
+				Expect(logs).To(ContainLines(ContainSubstring("PHP Buildpack")))
+				Expect(logs).To(ContainLines(ContainSubstring("Apache HTTP Server Buildpack")))
+				Expect(logs).To(ContainLines(ContainSubstring("PHP Web Buildpack")))
+				Expect(logs).To(ContainLines(ContainSubstring("PHP Composer Buildpack")))
+				Expect(logs).To(ContainLines(ContainSubstring("Procfile Buildpack")))
```

Comment:
How about adding an assertion that `web: procmgr ...` is printed? Procfile logs the start command it's setting